Python/SleepSort.py

```diff
@@ -0,0 +1,18 @@
+import threading
+from time import sleep
+
+def printSleepSort(i):
+    sleep(i) #Sleeps for i seconds
+    print(i) #Prints the value of i
+
+n = int(input("Enter the size of the array\n"))
+items = []
+
+print("Enter individual array elements in a new line.")
+for i in range(n):
+    items.append(int(input()))
+
+print("Printing the sorted array - ")
+for item in items:
+    th = threading.Thread(target=printSleepSort, args = (item,))
+    th.start()
```

Comment:
Add Sample I/O, Time and Space Complexity in a multiline comment.
Thank you!

Reply:
I am not adding the Space Complexity because for this particular problem that requires OS knowledge. I've added Sample I/O and Time complexity. Can you review them and tell me if any thing else is required?